### PR TITLE
Setup for RC build v0.26.0.rc.0

### PR DESCRIPTION
**Pull request by mattdurham**
#### PR Description

v0.26.0-rc0 release

#### PR Checklist

- [X] CHANGELOG updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ This document contains a historical list of changes between releases. Only
 changes that impact end-user behavior are listed; changes to documentation or
 internal API changes are not present.
 
-Main (unreleased)
------------------
+v0.26.0-rc.0 (2022-07-08)
+-------------------------
```
Comment (rfratto) [inline CHANGELOG.md:11]: Update to today?

**Review by rfratto — CHANGES_REQUESTED** [review]
You're missing updating references to the previous version which don't start with v (see grafanacloud-install.sh)

**Comment by rfratto** [comment]
I'd recommend creating a separate branch to cherry pick things into main to avoid needing to keep release-v0.26 mergable back upstream

**Comment by rlankfo** [comment]
I don't think it makes sense to merge the release candidate image tags/version updates to main. IMO they can just live on the release branch. Once the final release is cut, then the image tags can be merged into main.

**Pull request closed without merging**